csharp/extractor/Semmle.Extraction.CSharp/Entities/Property.cs

```diff
@@ -111,7 +111,7 @@ public override Microsoft.CodeAnalysis.Location FullLocation
 
         public static Property Create(Context cx, IPropertySymbol prop)
         {
-            bool isIndexer = prop.IsIndexer || prop.ExplicitInterfaceImplementations.Any(e => e.IsIndexer);
+            bool isIndexer = prop.IsIndexer || prop.ExplicitInterfaceImplementations.Any(e => e.IsIndexer) || prop.Parameters.Any();
```

Comment:
As discussed offline, simplify this to `prop.IsIndexer || prop.Parameters.Any()` and make sure we have a test for an indexer that explicitly implements an interface indexer.